spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -221,56 +271,37 @@ You can use the command `kubectl get all -l app=skipper` to verify that the depl
 +
 IMPORTANT: Specify the version of Spring Cloud Data Flow server that you want to deploy.
 +
-The deployment is defined in the `src/kubernetes/server/server-deployment.yaml` file. To control which version of Spring Cloud Data Flow server gets deployed, modify the tag used for the Docker image in the container specification, as follows:
+The deployment is defined in the `src/kubernetes/server/server-deployment.yaml` file.
+To control which version of Spring Cloud Data Flow server gets deployed, modify the tag used for the Docker image in the container specification, as follows:
 +
 ====
 [source,yaml]
 ----
     spec:
       containers:
       - name: scdf-server
-        image: springcloud/spring-cloud-dataflow-server-kubernetes:1.7.0.RELEASE   # <1>
+        image: springcloud/spring-cloud-dataflow-server-kubernetes:2.0.0.RELEASE   # <1>
         imagePullPolicy: Always
 ----
 
 <1> Change the version as you like. This document is based on the `{project-version}` release. The docker tag `latest` can be used for `BUILD-SNAPSHOT` releases.
 ====
 +
-[NOTE]
-=====
-To use Skipper, you must uncomment the following properties in `src/kubernetes/server/server-deployment.yaml`, under the `env:` section
-
-====
-[source,yaml,options=nowrap]
-----
-
- - name: SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI
-   value: 'http://${SKIPPER_SERVICE_HOST}/api'
- - name: SPRING_CLOUD_DATAFLOW_FEATURES_SKIPPER_ENABLED
-   value: 'true'
-----
-====
-=====
-+
-The Data Flow Server uses the https://github.com/fabric8io/kubernetes-client[Fabric8 Java client library] to connect to the Kubernetes cluster. We use environment variables to set the values needed when deploying the Data Flow server to Kubernetes. We also use the https://github.com/fabric8io/spring-cloud-kubernetes[Fabric8 Spring Cloud integration with Kubernetes library] to access the Kubernetes http://kubernetes.io/docs/user-guide/configmap/[ConfigMap] and http://kubernetes.io/docs/user-guide/secrets/[Secrets] settings.
-The ConfigMap settings for RabbitMQ are specified in the `src/kubernetes/server/server-config-rabbit.yaml` file and for Kafka in the `src/kubernetes/server/server-config-kafka.yaml` file. MySQL secrets are located in the `src/kubernetes/mysql/mysql-secrets.yaml` file. If you modified the password for MySQL, you should change it in the `src/kubernetes/mysql/mysql-secrets.yaml` file. Any secrets have to be provided in base64 encoding.
-+
-NOTE: We now configure the Data Flow server with file-based security, and the default user is 'user' with a password of 'password'. You should change these values in `src/kubernetes/server/server-config-rabbit.yaml` for RabbitMQ or `src/kubernetes/server/server-config-kafka.yaml` for Kafka.
+IMPORTANT: The Skipper service should be running and the `SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI` property in `src/kubernetes/server/server-deployment.yaml` should  points to that Skipper URI.
```

Comment:
`should  points` -> `should point`

Reply:
also remove that from "that Skipper URI"

Reply:
Changed it to:
```
IMPORTANT: The Skipper service should be running and the `SPRING_CLOUD_SKIPPER_CLIENT_SERVER_URI` property in `src/kubernetes/server/server-deployment.yaml` should  point to it.
```